### ISSUE TITLE
Callbacks doesn't work for accordion 

### ISSUE THREAD

**Issue by tilwinjoy** [angular-ui/bootstrap#5776]
I'm trying to attach callbacks with accordion. As per the documentation it extends from the collapse component which seems to support callbacks such as `expanding`,  `collapsing` etc.

 So I tried adding the following:

````html
<uib-accordion-group heading="first" collapsing="alert('collapsing')">
      first
</uib-accordion-group>
````

[plnkr demo](http://plnkr.co/edit/DusNhWbTb0v56UcAUxr9?p=preview)

Which I expect to work since accordion is an expansion of collapse. Is this a bug or am I doing something wrong..?

### Version of Angular, UIBS, and Bootstrap

Angular: 1.5.3

UIBS: 1.3.1

Bootstrap: 3.3.6

**Comment by pkozlowski-opensource** [comment]
> As per the documentation it extends from the collapse component which seems to support callbacks such as expanding, collapsing etc.

Accordion doesn't have same events, even if it build on top of `collapse`

**Comment by wesleycho** [comment]
This wouldn't be a bad feature request I think - those would be useful hooks to expose for some people.

**Comment by ericnewton76** [comment]
yes this would be nice... at least "expanding" event, so you can do some heavy lifting, like load a specific large image within container without loading all the containers' images.

tried to setup a jsfiddle but... http://jsfiddle.net/enorl76/pmrbhw04/

actually... if you put ng-click on it, you effectively get the expanding/expanded event.  See the fiddle.
